batchMonitorUpdate/batchUpdateReturnAll.py

```diff
@@ -0,0 +1,28 @@
+from datadog import initialize, api
+import datadog_api_client.v1
+from dateutil.parser import parse as dateutil_parser
+from datadog_api_client.v1 import ApiClient, ApiException, Configuration
```

Comment:
Seems like a lot of imports here that aren't actually being used?

Reply:
It might be good to include something like: https://github.com/DataDog/Miscellany/tree/master/get_all_services#get-started